tests/functional/testAddons.js

```diff
@@ -5,126 +5,199 @@
 const assert = require('assert');
 const queries = require('./queries.js');
 const vpn = require('./helper.js');
+const { SubscriptionDetails } = require('./servers/guardian_endpoints.js')
+const { startAndConnect } = require('./setupVpn.js')
 
 describe('Addons', function() {
   this.ctx.authenticationNeeded = true;
 
-  it('Empty addon index', async () => {
-    await vpn.resetAddons('01_empty_manifest');
-    await vpn.waitForCondition(async () => {
-      return parseInt(
-                 await vpn.getVPNProperty('VPNAddonManager', 'count'), 10) ===
-          0;
+  // it('Empty addon index', async () => {
```

Comment:
Why?

Reply:
Oh I am doing testing, the test is failing on CI.

Reply:
Ignore my last commit.